x-pack/plugins/translations/translations/ja-JP.json

```diff
@@ -8156,6 +8156,7 @@
     "xpack.enterpriseSearch.appSearch.engine.sampleEngineBadge": "サンプルエンジン",
     "xpack.enterpriseSearch.appSearch.engine.schema.conflicts": "スキーマ競合",
     "xpack.enterpriseSearch.appSearch.engine.schema.title": "スキーマ",
+    "xpack.enterpriseSearch.appSearch.engine.schema.unconfirmedFieldLabel": "最近追加された項目",
```

Comment:
These 'Recently added' translations were originally removed in https://github.com/elastic/kibana/pull/98955#discussion_r624038044 because we were simplifying the shared component. I removed them temporarily and have now re-added them back for this App Search specific usage (WS does not use this string at all).